source/features/i18n/translations/de.json

```diff
@@ -152,6 +152,8 @@
   "transaction": {
     "block": "Block",
     "confirmations": "Konfirmationen",
+    "deposit": "Anzahlung",
```

Comment:
@DominikGuzei This is just a Google Translate output, so let me know if it needs a tweak

Reply:
I spoke to Tommy (CF) and he said these should be: 

"deposit": "Kaution", 
"depositReclaim": "Kautionsrückforderung"